client/src/components/TripElement/TripEntry.js

```diff
@@ -86,61 +110,147 @@ export function TripEntry({ id, buttonClassName, trip }) {
         }}
         className={buttonClassName}
       >
-        {!isRenaming ? (
-          trip.tripName
-        ) : (
-          <input
-            ref={inputRef}
-            className='items-center bg-green-200/40 font-medium'
-            placeholder={trip.tripName}
-            value={tripName}
-            onChange={handleInputChange}
-          />
-        )}
+        <div className={`flex ${widthForTripEntry} justify-start space-x-2`}>
+          <PinDropOutlinedIcon className='align-baseline' />
+          {!isRenaming ? (
+            <div className={`${widthForTripEntry} truncate text-start`}>
+              {trip.tripName}
+            </div>
+          ) : (
+            <input
+              ref={inputRef}
+              className='w-[150px] items-center bg-green-200/40 font-medium'
+              placeholder={trip.tripName}
+              value={tripName}
+              onChange={handleInputChange}
+            />
+          )}
+        </div>
       </Button>
-      <div className='absolute right-0 top-0 flex flex-row'>
+    ),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
what was the issue with eslint here?

Reply:
eslint react-hooks always recommend to include every dependencies. in this case `widthForTripEntry` is recommended to be included in the array, but it is not necessary to re-render whenever `widthForTripEntry` is changed.

Reply:
gotcha. Anyway we can switch off this rule globally?

Reply:
we can switch it off. its better to keep it in for warnings